pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -169,6 +171,68 @@ profiles:
 				},
 			},
 		},
+		{
+			name: "v1beta2 with non-default global percentageOfNodesToScore",
+			data: []byte(`
+apiVersion: kubescheduler.config.k8s.io/v1beta2
+kind: KubeSchedulerConfiguration
+percentageOfNodesToScore: 10
+`),
+			wantProfiles: []config.KubeSchedulerProfile{
+				{
+					SchedulerName:            "default-scheduler",
+					PercentageOfNodesToScore: nil,
+					Plugins:                  defaults.PluginsV1beta2,
+					PluginConfig: []config.PluginConfig{
```

Comment:
`defaults.PluginConfigsV1beta2` here and similar for others below.

Reply:
Replaced it.  Other `PluginConfigs` cannot use `defaults.DefaultPluginConfigs` unless we modify `minCandidateNodesPercentage: 50` to `10` (default value) and/or `  bindTimeoutSeconds: 300` to `600` (default value). 

There might be a few places that can be improved. How about doing the clean up in another PR?

Reply:
yes, those sound tangential

Reply:
Thanks, added it to TODO list.